### DISCUSSION TITLE
How to remove "Variables" section in case it is not supported

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#215] [Q&A] [answered]
Hi all!

I have an self-hosted instance of the Console but I use neither Gitlab nor Vault to store my secret variables.
Therefore the "Variables" section is not usable in my context.

Is there a way to remove it from the "Company Overview" section of my self-hosted instance of the Console? 

Thanks

**Comment by fredmaggiowski** [comment, marked as answer]
Hi @Lucascanna currently there is no support for this; I guess something could be done if this is a problem for you 

@albertotessarotto @jacoposte this may be something we could address with the providers management: if there is no provider defined for Secret Management then there could be a placeholder in the section telling the user to add a provider that has the Secret Management capability
